suite-native/module-accounts/src/components/CoinPriceCard.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Box, Text, Card, RoundedIcon, Badge, Loader } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { networks } from '@suite-common/wallet-config';
+import { FiatAmountFormatter } from '@suite-native/formatters';
+import { AccountKey, TokenAddress } from '@suite-common/wallet-types';
+import { AccountsRootState, selectAccountNetworkSymbol } from '@suite-common/wallet-core';
+import { selectEthereumAccountToken } from '@suite-native/ethereum-tokens';
+
+import { useDayCoinPriceChange } from '../hooks/useDayCoinPriceChange';
+
+type CoinPriceCardProps = {
+    accountKey: AccountKey;
+    tokenContract?: TokenAddress;
+};
+
+type PriceChangeIndicatorProps = {
+    valuePercentageChange: number;
+};
+
+const cardStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItem: 'center',
+    padding: utils.spacings.medium,
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    borderRadius: utils.borders.radii.medium,
+}));
+
+const cardContentStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+}));
+
+const indicatorContainer = prepareNativeStyle(utils => ({
+    maxWidth: '40%',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    marginTop: utils.spacings.small / 4,
+}));
+
+const PriceChangeIndicator = ({ valuePercentageChange }: PriceChangeIndicatorProps) => {
+    const { applyStyle } = useNativeStyles();
+    const priceHasIncreased = valuePercentageChange >= 0;
+
+    const icon = priceHasIncreased ? 'arrowUp' : 'arrowDown';
+    const badgeVariant = priceHasIncreased ? 'green' : 'red';
+    const formattedPercentage = `${valuePercentageChange.toPrecision(3)} %`;
+
+    return (
+        <Box style={applyStyle(indicatorContainer)}>
+            <Text variant="label" color="textSubdued">
+                24h change
+            </Text>
+            <Box justifyContent="center" alignItems="center" flexDirection="row">
+                <Badge
+                    icon={icon}
+                    iconSize="extraSmall"
+                    size="medium"
+                    variant={badgeVariant}
+                    label={formattedPercentage}
+                />
+            </Box>
+        </Box>
+    );
+};
```

Comment:
Isn't this duplication of graph percentage indicator? Please unify into one component and move to atoms if possible